packages/alfa-rules/src/sia-r54/rule.ts

```diff
@@ -0,0 +1,54 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { isIgnored } from "../common/predicate/is-ignored";
+
+import { expectation } from "../common/expectation";
+import { Node } from "@siteimprove/alfa-aria";
```

Comment:
That should be moved up with the other global imports (first the global import (`@siteimprove/alfa-foo`) and next the local imports (`../foo`)).